<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phase_plane_widget</class>
 <widget class="QWidget" name="phase_plane_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="StrongBodyLabel" name="StrongBodyLabel">
     <property name="text">
      <string>Stata variables shown on axes and their range</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ComboBox" name="target_variable_x_cbb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Target Variable x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
       <property name="text">
        <string>start:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="tvx_start_edit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_3">
       <property name="text">
        <string>end:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="tvx_end_edit"/>
     </item>
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_6">
       <property name="text">
        <string>step:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="tvx_step_edit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="ComboBox" name="target_variable_y_cbb">
       <property name="text">
        <string>Target Variable y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_4">
       <property name="text">
        <string>start:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="tvy_start_edit"/>
     </item>
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_5">
       <property name="text">
        <string>end:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="tvy_end_edit"/>
     </item>
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
       <property name="text">
        <string>step:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="tvy_step_edit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_9">
       <property name="text">
        <string>Trajectory Initial Points x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="trajectory_points_x_edit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_11">
       <property name="text">
        <string>Trajectory Initial Points y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="trajectory_points_y_edit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_10">
       <property name="text">
        <string>Trajectory Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="trajectory_duration_edit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PrimaryPushButton" name="start_btn">
       <property name="text">
        <string>Start Phase Plane Analysis</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>220</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MplWidget" name="pp_mplWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="StrongBodyLabel" name="StrongBodyLabel_8">
       <property name="text">
        <string>Fixed Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="fixed_points_edit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>zjb.gui.widgets.mpl_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
